src/backend/optimizer/plan/aqumv.c

```diff
@@ -389,6 +393,7 @@ answer_query_using_materialized_views(PlannerInfo *root,
 		 */
 		preprocess_targetlist(subroot);
 		preprocess_aggrefs(subroot, (Node *) subroot->processed_tlist);
+		preprocess_aggrefs(subroot, mvQuery->havingQual);
```

Comment:
need `query->hasAggs` judge ?

Reply:
Good point, add `query->hasAggs` here is ok though `preprocess_aggrefs` could  handle NULL node well.

Reply:
Added.